tests/test_distro.py

```diff
@@ -1618,7 +1618,7 @@ def _bad_os_listdir(path='.'):
     raise OSError()
 
 
-@pytest.mark.skipIf(not IS_LINUX, reason='Irrelevant on non-linx')
+@pytest.mark.skipif(not IS_LINUX, reason='Irrelevant on non-linx')
```

Comment:
Might as well fix the "linux" typo while you're at it 😄

Reply:
It's done. Thank you!